dropwizard-client/src/main/java/io/dropwizard/client/ssl/TlsConfiguration.java

```diff
@@ -150,7 +156,18 @@ public void setSupportedProtocols(@Nullable List<String> supportedProtocols) {
         this.supportedProtocols = supportedProtocols;
     }
 
-    @ValidationMethod(message = "keyStorePassword should not be null or empty if keyStorePath not null")
+    @Nullable
+    @JsonProperty
+    public String getCertAlias() {
+		return certAlias;
+	}
```

Comment:
Yeah - the codeclimate checker pulled me up on that too. Am working through everything it reported.